WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -176,6 +176,11 @@ + (NSArray *)routes
   NSString *elementUUID = request.parameters[@"uuid"];
   XCUIElement *element = [elementCache elementForUUID:elementUUID];
   NSError *error = nil;
+  
+  //scrollelement to visiable but ignore errros happen during scroll
+  [element fb_scrollToVisibleWithError:&error];
```

Comment:
I did see that even though scroll was successful it failed with error.  I thought as long as click works fine we dont have to worry about scroll error.